apps/server/src/modules/workspaces/permission.ts

```diff
@@ -47,37 +47,38 @@ export class PermissionService {
 
   async isAccessible(ws: string, id: string, user?: string): Promise<boolean> {
     if (user) {
-      return await this.tryCheck(ws, user);
-    } else {
-      // check if this is a public workspace
-      const count = await this.prisma.workspace.count({
-        where: { id: ws, public: true },
-      });
-      if (count > 0) {
-        return true;
-      }
+      const hasPermission = await this.tryCheck(ws, user);
+      if (hasPermission) return true;
```

Comment:
```suggestion
      return this.tryCheck(ws, user)
```

Reply:
if user not have full workspace permission, we need continue check sub page permission